pkg/distributor/distributor_ingest_storage_test.go

```diff
@@ -70,16 +75,16 @@ func TestDistributor_Push_ShouldSupportIngestStorage(t *testing.T) {
 		"should shard series across all partitions when shuffle sharding is disabled": {
 			shardSize: 0,
 			expectedSeriesByPartition: map[int32][]string{
-				0: {"series_one", "series_three", "series_four"},
```

Comment:
Note to reviewers: I've added an utility function to extrapolate series written to Kafka and it now returns **sorted** metric names.